sdk/src/main/kotlin/com/processout/sdk/di/NetworkGraph.kt

```diff
@@ -32,10 +33,10 @@ internal class NetworkGraphImpl(configuration: NetworkConfiguration) : NetworkGr
             .addInterceptor(BasicAuthInterceptor(configuration.projectId, configuration.privateKey))
             .addInterceptor(UserAgentInterceptor(configuration.application, configuration.sdkVersion))
             .apply {
-                if (BuildConfig.DEBUG) {
-                    addInterceptor(HttpLoggingInterceptor().apply {
-                        level = HttpLoggingInterceptor.Level.BODY
-                    })
+                if (configuration.debug) {
+                    addInterceptor(HttpLoggingInterceptor { message ->
+                        Log.println(Log.DEBUG, BuildConfig.LIBRARY_PACKAGE_NAME, message)
```

Comment:
Can we use our own logger here `POLogger.debug(...)`? It should allow to avoid passing tags explicitly.

Reply:
It will add `[NetworkGraphImpl:40]` before each http log line, which is extra noise in this context.

Reply:
Hmm, solution could be to add parameter to POLogger calls that would allow to opt out from including caller metadata in such use case.

Reply:
Decided to use POLogger and keep class name. Actually it will be useful to filter out http logs from debug level when developer needs to see debug logs from other classes, but not necessary interested in http logs.
`package:mine tag:com.processout.sdk level:debug -message:NetworkGraphImpl`